<?xml version="1.0"?>
<launch>
    <arg name="config_file" default="zed.yaml" />
    
    <node pkg="lvi_calib" type="apriltag_test_node" name="apriltag_test_node" output="screen">
      <param name="config_file"  type="string" value="$(find lvi_calib)/cfg/$(arg config_file)" />
    </node>

</launch>

